extensions/amp-story/0.1/amp-story.js

```diff
@@ -324,6 +395,13 @@ export class AmpStory extends AMP.BaseElement {
     const targetPage = this.getPageById_(targetPageId);
     const pageIndex = this.getPageIndex(targetPage);
 
+    if (this.prevButton_) {
+      this.prevButton_.classList.toggle(
+          'i-amphtml-story-button-move-hidden', pageIndex === 0);
+    }
+
+    this.updateBackground_(targetPage.element);
```

Comment:
I see this here in `layoutCallback` for the initial page, but how does this get updated when you navigate?

Reply:
`this.updateBackground_` is called in `switchTo`, which is called on navigate.

Reply:
oh whoops, I didn't read this right... my bad.